bigchaindb/web/views/parameters.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+
+def valid_txid(txid):
+    if re.match('^[a-fA-F0-9]{64}$', txid):
+        return txid.lower()
+    raise ValueError("Invalid hash")
+
+
+def valid_bool(val):
```

Comment:
It's used in https://github.com/bigchaindb/bigchaindb/pull/1066/files#diff-96761b3426cf97be2817aa6a0ebff3f7R18 which actually includes all of the code in this PR.